.github/labeler.yml

```diff
@@ -0,0 +1,23 @@
+# https://github.com/actions/labeler#common-examples
+# Generated from https://github.com/rapidsai/cudf/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/cudf/labels
+
+cuDF (Python):
+  - 'python/**'
+  - 'notebooks/**'
+  
+cpp:
+  - 'cpp/**'
+
+CMake:
+  - '**/CMakeLists.txt'
+  - '**/cmake/**'
+  
+cuDF (Java):
+  - 'java/**'
+
+ci:
```

Comment:
FYI this should be `gpuCI` to match the [label](https://github.com/rapidsai/cudf/labels?q=gpuci)